boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,821 @@
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+#define exit_ok_or_return(code) \
```

Comment:
Should be uppercase

Reply:
No he didn't but I stuck it in the .c since I don't see us using it anywhere else currently. I think we should leave it here for now and move it to App_Efuse.h if we need it there as well later?